can-bus/json/BMS/BMS_tx.json

```diff
@@ -14,5 +24,18 @@
                 "bits": 1
             }
         }
+    },
+    "BMS_Contactors": {
+        "msg_id": 103,
+        "cycle_time": 1000,
+        "description": "Signals related to the contactors.",
+        "signals": {
+            "AirPositiveState": {
+                "enum": "AirState"
+            },
+            "AirNegativeState": {
+                "enum": "AirState"
+            }
+        }
```

Comment:
Again, I should keep this consistent on my end. I like the name BMS_Contactors as opposed to air states. My msg-id for this is 112 tho. Idk, we could also just deal with the merge conflicts when they arise, I feel like they're inevitable.